src/please.go

```diff
@@ -1449,6 +1449,8 @@ func execute(command string) int {
 		}()
 	}
 	defer worker.StopAll()
+
+	log.Debugf("plz command: %v", strings.Join(os.Args, " "))
```

Comment:
Needed?

Reply:
Bit of a bonus change. I think this could be useful. I found it hard to figure out which command the log files belonged to when debugging the cards exporter build.